test/functional/data.spec.js

```diff
@@ -60,11 +61,30 @@ describe('swag-for-dev', function () {
 						});
 					}
 
-					it(opportunity.name + ' has a valid image', function () {
+					it(opportunity.name + ' has a valid image URL', function () {
```

Comment:
From my perspective, there's no need to check if the image URL is valid since we're checking the mime type of the image - by checking the mime type of the image, we're implicitly validating that the image also exists

Reply:
I added new test for mime-type without modifying existing ones. Just changed test description text.

I agree to your point. Downloading and checking an image means implicitly checking for valid URL.